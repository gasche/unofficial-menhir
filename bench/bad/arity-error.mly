%token FOO BAR

%%

a(X):
  x = X { x }

foo:
  a(FOO) a(BAR,BAR) { () }
