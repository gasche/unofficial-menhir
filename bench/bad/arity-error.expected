File "arity-error.mly", line 9, characters 9-10:
Error: does this symbol expect 1 or 2 arguments?
